apps/upgrade-examples-v8-v9/src/stories/CommandBar.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { CommandButton, IButtonProps, Icon, IIconProps } from '@fluentui/react';
+import { Button, webLightTheme, FluentProvider, makeStyles } from '@fluentui/react-components';
+import { CommandButtonShim } from '../shims/Button';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto',
+    columnGap: '20px',
+    justifyContent: 'center',
+    justifyItems: 'center',
+    alignContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+const addIcon: IIconProps = { iconName: 'Add' };
+
+export const CommandBar = (props: IButtonProps) => {
```

Comment:
Looks like this file is a duplicate of `Command.stories.tsx` and isn't being used

Reply:
heh dont know why this was this duplicated 😆